Docs: Clarifies use of "config" & "monitor" nodes

matthiashh opened the pull request:
These were the two things I got wrong first:

1. "config" does not appear in the palette and is configured inside other nodes:
I almost opened an issue for this, because I could not find the "config" node in the palette. I had other modules in the past that required to place a special "config" node without connections anywhere in the flow.

2. "monitor" provides all updates, not just if a device is added or removed:
First I though that this only sends information when devices are newly added (updated) to the network or removed from it. It turned out that it provides all status updates though.

matthiashh committed:
Docs: Clarifies use of "config" & "monitor" nodes

- "config" does not appear in the palette and is configured inside other nodes
- "monitor" provides all updates, not just if a device is added or removed

mjwwit commented:
Thanks for this!

> 1. "config" does not appear in the palette and is configured inside other nodes:
>    I almost opened an issue for this, because I could not find the "config" node in the palette. I had other modules in the past that required to place a special "config" node without connections anywhere in the flow.

This is actually a pattern that is very common for Node-RED plugins and even built-in nodes. So-called configuration nodes are never visible in the palette, because they can not be added separately to your flows. I'll still include the addition you made to the docs however, as it can't hurt.

mjwwit commented on README.md:
```diff
@@ -51,7 +53,7 @@ If both are specified, the node will pick the action (on or off) from the messag
 
 ### tradfri-monitor
-This node will send messages when a device or group is updated or removed. The message will contain all available details for the device or group. Due to how this system works these update messages are also sent when reconnecting to the gateway after a flow deploy.
+This node will send messages when a device or group is updated or removed. Examples for updates are buttons presses, change of brightness or a switch being turned on. The message will contain all available details for the device or group. Due to how this system works these update messages are also sent when reconnecting to the gateway after a flow deploy.
```
I'm actually still unsure if I can build support for switches being toggled or buttons being pressed. So updates are usually state changes of any connected devices (lights being turned on, brightness being changed, blinds opening or closing, etc.), but not the actual buttons being clicked.
I'll need to finish work on #21 before this would work.

matthiashh replied:
Fair point. What I meant is that an event is triggered when a button is pressed, but you are right that the information "which button" is not exposed yet. I'll remove it, so people do not have wrong expectations.

matthiashh committed:
Docs: Removes "button presses" from example events

While an event is created when a button is pressed, the information about the button is not transmitted, see #21.

matthiashh committed:
Docs: Clarifies which devices support status updates

Not all devices are supported yet, but it's not mentioned in the docs yet. See e.g. #21 and #23.

mjwwit commented:
Very nice, thank you! I'll merge this.
